rpm/README.md

```diff
@@ -0,0 +1,35 @@
+# RPM packaging
+
+The template in fingertip.spec contains template for packaging fingertip in
+Fedora and RHEL. As fingertip does not provide any releases yet, the latest
+git version is packaged (`git_commit` macro) and version is considered as
+`0.git_date`, where `git_date` is the date of package creation.
+
+# Content
+
+Aside of the spec file, there are additional files. One of them is `fingertip.in`
+wrapper, which is used to run `fingertip` command by the end users. It takes
+care of correct location of the python library and initial setup (creation of
+CoW-enabled filesystem in a file under home directory after user confirmation).
+
+What is missing is `systemd.timer` that would run cleanup of caches automatically.
+
+# Updating package
+
+For pulling new changes from upstrem, update `git_commit` and `git_date`
```

Comment:
'upstream'

Reply:
Fixed in current version. Thanks for review.